ledger/ledger-api-common/src/main/scala/com/digitalasset/ledger/api/tls/DecryptionParameters.scala

```diff
@@ -39,14 +39,24 @@ case class DecryptionParameters(
     transformation.split("/")(0)
   }
 
+  private def decodeBase64OrGetVerbatim(encrypted: Array[Byte]): Array[Byte] = {
```

Comment:
Shall we document what is the intended behavior of this function?

Reply:
I guess this is relevant: 
>  All line separators or other characters not found in the base64 alphabet table are ignored in decoding operation.
from MIME's section of `java.util.Base64` javadoc

Reply:
Thanks for the feedback. I will create a PR against the main branch for this